### DISCUSSION TITLE
How to connect with the API

### DISCUSSION THREAD

**Discussion by maaper** [docusealco/docuseal#78] [Q&A]
Dear all, 
I was trying on connected with the API or getting one webhook with the information of every signed document received. 
My idea will be to integrate with N8N to keep completing a list of the receiving forms. 
Thanks in advance.

**Comment by maaper** [comment]
Now I see the way after the last update, thanks!!!

**Comment by ovizii** [comment]
It is nice that you see the way to do it, but would you mind sharing it with us too?

**Reply by maaper** [reply]
Hi, if you are running on docker, just update your docker to :latest release and you will see on SETTINGS both API and WEBHOOK options.
Thanks to that I could integrate docuseal with my N8N workflows.

![image](https://github.com/docusealco/docuseal/assets/56989894/d4ae8bd7-f04a-475e-82d5-2388e06cadb1)

**Reply by ovizii** [reply]
Thanks, I didn't realize that with some update those settings had appeared :-)
